### PR TITLE
Add LL/DDI/ADV/BV-27-C test case

### PR DESCRIPTION
**Pull request by Tronil**
Adds the LL/DDI/ADV/BV-27-C test case to ll_verification.py

Signed-off-by: Troels Nilsson [trnn@demant.com](mailto:trnn@demant.com)

### CODE REVIEW

File: src/tests/ll_verification.py [hunk 1]
```diff
@@ -723,6 +723,182 @@ def ll_ddi_adv_bv_21_c(transport, upperTester, lowerTester, trace):
...
+                if len(remainingAdvData) <= maxFragmentSize:
+                    op = FragmentOperation.LAST_FRAGMENT
+                else:
+                    op = FragmentOperation.INTERMEDIATE_FRAGMENT
+            endIndex = maxFragmentSize if len(remainingAdvData) >= maxFragmentSize else len(remainingAdvData)
+            status = le_set_extended_advertising_data(transport, upperTester, Handle, op, 0x00, remainingAdvData[:endIndex], 100)
+            if status != 0:
+                return False
+            remainingAdvData = remainingAdvData[endIndex:]
+            firstFragment = False
```
[the remark this answers is not in the document]
Comment (Tronil) [reply src/tests/ll_verification.py:798]: Yes that would make sense

File: src/tests/ll_verification.py [hunk 2]
```diff
@@ -783,27 +941,9 @@ def do_ll_ddi_adv_bv_47_49_c(transport, upperTester, lowerTester, trace, packets
...
-                        op = FragmentOperation.LAST_FRAGMENT
-                    else:
-                        op = FragmentOperation.INTERMEDIATE_FRAGMENT
-                endIndex = maxFragmentSize if len(remainingAdvData) >= maxFragmentSize else len(remainingAdvData)
-                status = le_set_extended_advertising_data(transport, upperTester, Handle, op, round.FragmentPref, remainingAdvData[:endIndex], 100)
-                if status != 0:
-                    return False
-                remainingAdvData = remainingAdvData[endIndex:]
-                firstFragment = False
+            if not set_complete_ext_adv_data(transport, upperTester, Handle, advData):
```
Comment (wopu-ot) [inline src/tests/ll_verification.py:945]: I think the `round.fragmentPref` got lost in the refactoring

**Review by wopu-ot — CHANGES_REQUESTED** [review]
LGTM apart from the issue introduced in the refactoring

File: src/tests/ll_verification.py (hunk 2 above)
Comment (Tronil) [reply src/tests/ll_verification.py]: Sure did, fixed now

**Review by wopu-ot — APPROVED** [review]

**Pull request merged**